src/main/groovy/org/mockito/release/notes/model/Commit.java

```diff
@@ -8,6 +8,11 @@
 public interface Commit {
 
     /**
+     * Commit identifier. For git it would be 'hash' (SHA-1)
+     */
+    String getCommitId();
```

Comment:
Yes, it's not really needed at the moment. I added it at the begin of my implementation, because I thought I need it in some GitHub API call's. Finally I found another way to solve the problem. It can be useful in the future for logging / debugging when something goes wrong.